enhancements/hypershift/performanceprofilecontroller-node-tuning.md

```diff
@@ -0,0 +1,328 @@
+
+api-approvers:
+  - None
+
+tracking-link:
+  - https://issues.redhat.com/browse/PSAP-742
+  
+see-also:
+  - "/enhancements/hypershift/node-tuning.md"
+
+creation-date: 2022-09-19
+last-updated: 2022-09-19
+---
+
+# Performance Profile Controller Adaptation to Hypershift
+
+## Summary
+
+This enhancement describes how the [Performance Profile Controller](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/performance_controller.md), previously known as Performance add-on operator,
+will run in Hypershift's managed control plane.
+
+This is part of Phase-3 for Node Tunning Operator enablement and so it depends on the two previous phases.
+
+## Motivation
+
+See the HyperShift [project goals](https://hypershift-docs.netlify.app/reference/goals-and-design-invariants/).
+These goals should not come at the cost of giving up any existing OpenShift features.
+
+In standalone OpenShift Performance Profile Controller, as part of Node Tuning Operator, helps in optimizes OpenShift clusters for applications sensitive to cpu and network latency
+by letting the user to create a more comprehensive and unify API from where the controller extract the appropiate parameters,
+creates the kubernetes objects and apply them to the worker nodes in the cluster.
+
+### User Stories
+
+[See](/enhancements/hypershift/node-tuning.md#user-stories)
+
+### Goals
+
+- Handle the PerformanceProfile objects in the hosted control plane namespaces while applying the output manifests to the appropiate worker nodes in the hosted cluster.
+
+### Non-Goals
+
+- N/A
+
+## Proposal
+
+The Performance Profile Controller, being part of the Node Tuning Operator, will run in the hosted control plane namespace, alongside several other control plane components.
+
+Performance Profile Controller optimizes OpenShift clusters for applications sensitive to cpu and network.
+More info [here](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/performance_controller.md)
+
+[PerformanceProfile CRD](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/performance_profile.md) is the API for Performance Profile Controller. Using a PerformanceProfile as input Performance Profile Controller currently generates four different objects:
+
+- machineconfiguration.openshift.io/v1/MachineConfiguration
+- machineconfiguration.openshift.io/v1/KubeletConfig
+- tuned.openshift.io/v1/Tuned
+- node.k8s.io/v1/RuntimeClass
+
+In a Standalone deployment this CRs are generated by Performance Profile Controller and then reconciled by the appropriate controllers/operators running in the same cluster.
+In a Hypershift deployment we need to change how these CRs are created.
+
+### Workflow Description
+1. Cluster Service Consumer will create PerformanceProfile objects inside of ConfigMaps, under the tag `tuned`, and reference these ConfigMaps in the `spec.tunedConfig` field of the NodePool API.
```

Comment:
> under the tag `tuned`

It may be clearer to say "using a key named `tuned`"

Reply:
Done, thanks for your review.